src/applications/claims-status/tests/components/FilesPage.unit.spec.jsx

```diff
@@ -270,175 +195,4 @@ describe('<FilesPage>', () => {
-      },
-    };
-
-    const tree = SkinDeep.shallowRender(
-      <FilesPage params={{ id: 2 }} claim={claim} />,
-    );
-    const content = tree.dive(['FilesPageContent']);
-    expect(content.everySubTree('AskVAToDecide')).not.to.be.empty;
-  });
-
-  it('should display turned in docs', () => {
-    const claim = {
-      attributes: {
-        eventsTimeline: [
-          {
-            type: 'received_from_you_list',
-            documents: [
-              {
-                filename: 'Filename',
-              },
-            ],
-            trackedItemId: 2,
-            status: 'ACCEPTED',
-          },
-        ],
-      },
-    };
-
-    const tree = SkinDeep.shallowRender(<FilesPage claim={claim} />);
-    const content = tree.dive(['FilesPageContent']);
-    expect(content.everySubTree('SubmittedTrackedItem').length).to.equal(1);
-  });
-
-  it('should display additional evidence docs', () => {
-    const claim = {
-      attributes: {
-        eventsTimeline: [
-          {
-            filename: 'Filename',
-            fileType: 'Testing',
-            type: 'other_documents_list',
-          },
-        ],
-      },
-    };
-
-    const tree = SkinDeep.shallowRender(<FilesPage claim={claim} />);
-    const content = tree.dive(['FilesPageContent']);
-    expect(content.everySubTree('AdditionalEvidenceItem').length).to.equal(1);
-  });
-
-  it('should show never received docs as tracked items', () => {
-    const claim = {
-      attributes: {
-        eventsTimeline: [
-          {
-            type: 'never_received_from_you_list',
-            documents: [
-              {
-                filename: 'Filename',
-              },
-            ],
-            trackedItemId: 2,
-            status: 'ACCEPTED',
-          },
-          {
-            type: 'never_received_from_others_list',
-            documents: [
-              {
-                filename: 'Filename',
-              },
-            ],
-            trackedItemId: 3,
-            status: 'NEEDED',
-          },
-        ],
-      },
-    };
-
-    const tree = SkinDeep.shallowRender(<FilesPage claim={claim} />);
-    const content = tree.dive(['FilesPageContent']);
-    expect(content.everySubTree('SubmittedTrackedItem').length).to.equal(2);
-    expect(content.everySubTree('AdditionalEvidenceItem')).to.be.empty;
-  });
-
-  it('should clear alert', () => {
-    const claim = {
-      attributes: {
-        eventsTimeline: [],
-      },
-    };
-    const clearNotification = sinon.spy();
-    const message = {
-      title: 'Test',
-      body: 'Test',
-    };
-
-    const tree = SkinDeep.shallowRender(
-      <FilesPage
-        clearNotification={clearNotification}
-        message={message}
-        claim={claim}
-      />,
-    );
-    expect(clearNotification.called).to.be.false;
-    tree.subTree('ClaimDetailLayout').props.clearNotification();
-    expect(clearNotification.called).to.be.true;
-  });
-
-  it('should clear notification when leaving', () => {
```

Comment:
I think we want this test we just need to update the claim for lighthouse

Reply:
Done